app/api/services/briefs.py

```diff
@@ -0,0 +1,21 @@
+from app.api.helpers import Service
+from app import db
+from app.models import Brief, BriefResponse
+
+
+class BriefsService(Service):
+    __model__ = Brief
+
+    def __init__(self, *args, **kwargs):
+            super(BriefsService, self).__init__(*args, **kwargs)
```

Comment:
do we do linting on py files?  isn't that an extra indent?

Reply:
because the pep8 rule we are using is `must be multiple of 4` but you're right